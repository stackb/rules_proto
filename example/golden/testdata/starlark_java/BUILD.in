# "proto_rule" instantiates the proto_compile rule
# gazelle:proto_rule proto_compile implementation stackb:rules_proto:proto_compile
# gazelle:proto_rule java_library implementation lib/rules.star
# gazelle:proto_rule java_library deps @com_google_protobuf//:protobuf_java
# gazelle:proto_rule java_library deps @com_google_protobuf//java/core
# gazelle:proto_rule java_library visibility //visibility:public

# "proto_plugin" instantiates the starlark java plugin
# gazelle:proto_plugin java implementation lib/plugins.star

# "proto_language" binds the rule(s) and plugin(s) together
# gazelle:proto_language java rule proto_compile
# gazelle:proto_language java rule java_library
# gazelle:proto_language java plugin java
