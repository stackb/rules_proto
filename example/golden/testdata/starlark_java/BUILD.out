# gazelle:proto_rule proto_compile implementation stackb:rules_proto:proto_compile
# gazelle:proto_rule java_wrapper implementation lib/rules.star%java_wrapper
# gazelle:proto_rule java_library implementation lib/rules.star%java_library
# gazelle:proto_rule java_library deps @com_google_protobuf//:protobuf_java
# gazelle:proto_rule java_library deps @com_google_protobuf//java/core
# gazelle:proto_rule java_library visibility //visibility:public
# gazelle:proto_plugin java implementation lib/plugins.star%java
# gazelle:proto_language java rule proto_compile
# gazelle:proto_language java rule java_library
# gazelle:proto_language java rule java_wrapper
# gazelle:proto_language java plugin java
# gazelle:proto_language java enabled true
